metricflow/dataflow/dataflow_plan.py

```diff
@@ -343,10 +344,23 @@ class AggregatedMeasuresOutput(Generic[SourceDataSetT], BaseOutput[SourceDataSet
 
 
 class AggregateMeasuresNode(Generic[SourceDataSetT], AggregatedMeasuresOutput[SourceDataSetT]):
-    """A node that aggregates the measures by the associated group by elements."""
+    """A node that aggregates the measures by the associated group by elements.
 
-    def __init__(self, parent_node: BaseOutput) -> None:  # noqa: D
+    In the event that one or more of the aggregated input measures has an alias assigned to it, any output query
```

Comment:
Given the alias behavior, I'm thinking if we should rename this node. Can't come up with a name that isn't awkward though.

Reply:
I can't find this comment anywhere. WTF GitHub?

Anyway, yeah, it's tough, because MaybeAliasedAggregateMeasuresNode is super awkward, and AliasedAggregateMeasuresNode is also awkward and not exactly correct.

My personal preference is to find a way to have a type bound to the relevant operations so that the aliased or not behavior is either invisible to the caller (because they just call the common interface method and get the right value) or else is breaking in the type system (because anywhere they expect a non-aliased thing they cannot accept one of these), but I couldn't figure out how to make that happen without a pretty major set of changes. Something for us to powwow about later, perhaps.